GooglePlayInstant/Tests/Editor/QuickDeploy/AssetBundleVerifierTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using GooglePlayInstant.Editor.QuickDeploy;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.TestTools;
+
+namespace GooglePlayInstant.Tests.Editor.QuickDeploy
+{
+    /// <summary>
+    /// Contains tests for each of AssetBundleVerifier error states.
+    /// </summary>
+    [TestFixture]
+    public class AssetBundleVerifierTest
+    {
+        [UnityTest]
+        public IEnumerator TestDestinationError()
+        {
+            var window = (AssetBundleVerifierWindow) EditorWindow.GetWindow(typeof(AssetBundleVerifierWindow), true,
+                "Play Instant AssetBundle Verify");
+
+            while (window == null)
```

Comment:
I don't understand why this block is needed:
- Why isn't window immediately set to a non-null value the by the statement above?
- If you remove this while loop does the test not work?
- Can you link to some documentation explaining this?

Reply:
Sorry, I think I miscorrectly diagnosed a problem with the tests to the window object not rendering in time; I was getting NullPointerExceptions previously without the use of this specific block. I did all the changes, and retried it again, but it seems to be working. Not sure what the issue was before. Resolved.